pulp_rpm/tests/functional/api/test_modular.py

```diff
@@ -47,3 +47,25 @@ def test_is_modular_flag(
     # assert all package from modular repo is marked as modular
     for pkg in get_content(repo_modular.to_dict())[RPM_PACKAGE_CONTENT_NAME]:
         assert pkg["is_modular"] is True
+
+
+def test_profiles_and_description_info(
```

Comment:
@pavelpicka Don't make it so specific, just test the read operation generally including all the fields that are supposed to be there.

Reply:
@dralley changed to check all fields which should be present